src/library_pthread.js

```diff
@@ -408,6 +408,7 @@ var LibraryPThread = {
               if (detached) {
                 PThread.returnWorkerToPool(worker);
               }
+#if EXIT_RUNTIME // If building with -s EXIT_RUNTIME=0, no thread will post this message, so don't even compile it in.
```

Comment:
What about if the application actually calls "exit()"?   Or does `-s EXIT_RUNTIME=0` keep the program alive in that case too?

Reply:
If you look at the implementation of `exit()`, you can see that it is a no-op in `-s EXIT_RUNTIME=0` builds. (Also calling `exit()` never posted a `exitProcess` message)